about.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>WAGTASTIC</title>
+  <link rel="stylesheet" type="text/css" href="group1.css">
+
+</head>
+<body>
+
+   <!-- Header  -->
+
+  <header>
+    <nav>
+      <h1>WAGTASTIC</h1>
+      <ul>
+        <li><a href="http://www.Wagtastic.com/Products" title="Products">Products</a></li>
+        <li><a href="http://www.Wagtastic.com/About_Us" title="About_Us">About Us</a></li>
+        <li><a href="http://www.Wagtastic.com/Contact" title="Contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class= "about-content">
+    <img src="https://pixabay.com/static/uploads/photo/2012/12/24/08/40/dog-72333_960_720.jpg" alt="pup!">
```

Comment:
p tags are indented an extra time. Second closing p tag has wandered away from the line.